### ISSUE TITLE
[FEATURE] Add UMU-Proton to ProtonPlus

### ISSUE THREAD

**Issue by YoreonX** [Vysp3r/ProtonPlus#269]
Glorious Eggroll has released a Universal Proton, that is going to be the future of Third Party Proton support which is why Wine-GE no longer gets updates.

No need to remove Wine-GE, but add UMU support as a Runner for the existing Tabs: Steam, Lutris, Bottles

I assume UMU works the same as Proton-GE, if this is not accurate just inform me &  close the thread.

**Comment by YoreonX** [comment]
Yep I just added UMU-Proton-9.0.3 manually to both Lutris/Bottles Flatpak and it works perfectly.

Here is the Open Wine Componets Github: https://github.com/Open-Wine-Components/umu-proton

**Comment by Vysp3r** [comment]
I am currently doing my basic military training so I will not be available for a while, but once I can I will do it for sure.

**Comment by Arcitec** [comment]
UMU is a really cool project. It's made by Glorious Eggroll and the authors of Lutris, Bottles and one of the biggest Heroic contributors. The goal is to run the real Proton and Steam Linux Runtime outside of Steam. So it makes sense to add support for it. But we have to wait for Vysp3r to come back home, which will take a month or two.

**Comment by polluxau** [comment]
There is a great video about this with glorious doing his presentation about umu at Ubuntu summit 

It's around 1 hour into the Livestream :)

https://www.youtube.com/live/byPpJW5l6pg?si=VclUXS4YWy7peDVq

I'm very excited for this to be implemented by every type of wine game manager/launcher :>

**Comment by Vysp3r** [comment]
I've successfully added Proton support in Lutris without having to rely on Steam.
It will be included in the next update.
I will be busy the next two weeks, but after that I should have free time to continue since my training will be done.
